integration_tests/CMakeLists.txt

```
@@ -1856,12 +1856,12 @@ RUN(NAME openmp_32 LABELS gfortran llvm_omp GFORTRAN_ARGS -fopenmp)
 RUN(NAME openmp_33 LABELS gfortran llvm_omp NO_EXPERIMENTAL_SIMPLIFIER GFORTRAN_ARGS -fopenmp) # mandelbrot
 RUN(NAME openmp_34 LABELS gfortran llvm_omp GFORTRAN_ARGS -fopenmp) # Ensure uniform load distribution over threads
 RUN(NAME openmp_35 LABELS gfortran llvm_omp GFORTRAN_ARGS -fopenmp)
-RUN(NAME openmp_36 LABELS gfortran GFORTRAN_ARGS -fopenmp)
+RUN(NAME openmp_36 LABELS gfortran GFORTRAN_ARGS -fopenmp llvm)
```

```suggestion
RUN(NAME openmp_36 LABELS gfortran llvm GFORTRAN_ARGS -fopenmp)
```

It will take `llvm` as `GFORTRAN_ARGS`

---

Thanks!!